packages/dd-trace/src/exporters/common/writer.js

```diff
@@ -24,7 +24,10 @@ class Writer {
   }
 
   append (spans) {
-    if (!request.writable) return
+    if (!request.writable) {
+      log.debug(`Maximum number of active requests reached. Trace discarded: ${JSON.stringify(spans)}`)
```

Comment:
The message should be wrapped in a function to avoid calling `stringify` outside of debug mode.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/2296/commits/d6c71deb7dd8087604f174b0eed972857ae25e30